src/Files.Shared/Services/IResourcesService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Drawing;
+
+namespace Files.Shared.Services;
+
+/// <summary>
+/// Contains methods related to modifying various app theme resources
+/// </summary>
+public interface IResourcesService
```

Comment:
Just a note -- this interface doesn't seem like it belongs to Files.Shared. Try to put it in the backend. Also, it doesn't really look abstract enough - try to be platform independent 🙂